builtin/game/misc.lua

```diff
@@ -166,6 +166,18 @@ function core.setting_get_pos(name)
 	return core.string_to_pos(value)
 end
 
+function core.get_world_limits()
+	local absolute_limit = 31000
```

Comment:
IIRC (from studying the code some time ago, while implementing this PR), 31000 is an absolute limit. No block will be generated that is beyond that limit. Ever. Actually, no _chunk_ will be generated that countains a block that exceeds the limit, or even that contains a block that is adjacent to the limit: when generating a chunk, a volume that is 1 block larger in all directions is processed; IIRC, if just one node in that volume being processed is beyond 31000, then the chunk won't be generated.

So basicallly, between the most distant node of a world that will be generated and the limit of 31000 (or whatever it is), there should always be at least one block of non-generated world.

IIRC, with a chunk size of 5, the actual limit is about 30912 in one direction, and at most 15 or maybe 31 more in the other.